consultation_analyser/consultations/views.py

```diff
@@ -1,12 +1,22 @@
 from django.shortcuts import render
 from django.http import HttpRequest
-from .models import Question
+from . import models
 
 
 def home(request: HttpRequest):
     return render(request, "home.html")
 
 
 def show_question(request: HttpRequest, slug: str):
-    context = {"question": Question.objects.filter(slug=slug).first()}
+    context = {"question": models.Question.objects.filter(slug=slug).first()}
     return render(request, "show_question.html", context)
+
+
+def question_summary(request: HttpRequest, consultation_slug: str, section_slug: str, question_slug: str):
```

Comment:
I don't think this is a different view to the one above?

Reply:
Now renamed to `show_question` and the other view removed.